integration-tests/__tests__/__snapshots__/globals.test.js.snap

```diff
@@ -28,7 +28,7 @@ exports[`cannot test with no implementation 1`] = `
       1 | 
       2 |     it('it', () => {});
     > 3 |     it('it, no implementation');
-        | ^
+        |     ^
```

Comment:
Probably we can kill https://github.com/facebook/jest/blob/master/integration-tests/__tests__/globals.test.js#L31-L35?

Reply:
done